pages/service/[id].tsx

```diff
@@ -330,6 +367,29 @@ export const ServicePage = ({ serviceData }: ServicePageProps): JSX.Element => {
         </ContactList>
       </Section>
 
+      <Section divider={'transparent'}>
+        <Heading as="h3">Review</Heading>
+        <FiveStar
+          currentRating={reviewState.rating}
+          onChange={onRatingChange}
+        />
+        <Checkbox
+          label={'I have attended this service'}
+          checked={reviewState.hasUsedService}
+          onChange={onUsedServiceChange}
+        />
+        <span>Leave a review</span>
+        <TextInput
```

Comment:
textarea can take a `rows` attribute that guarantees the height matches the text inside exactly